login-screen/app.json

```diff
@@ -1,35 +0,0 @@
-{
-  "expo": {
-    "facebookScheme": "fb1047121222092614",
-    "facebookAppId": "1047121222092614",
-    "facebookDisplayName": "instmobile",
-    "android": {
-      "package": "io.instamobile.login"
-    },
-    "name": "login-screen",
-    "description": "Login Screen implementation by https://www.instamobile.io and https://www.iosapptemplates.com .",
```

Comment:
we need to keep the same description, package name, name, slug, in the new app.json etc so we can push this to expo